simplefs/simplefs_test.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package simplefs
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/keybase1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestListLocal(t *testing.T) {
+	ctx := context.Background()
+	sfs := &SimpleFS{}
+
+	// make a temp local directory + files we will clean up later
+	tempdir, err := ioutil.TempDir("", "simpleFstest")
+	defer os.RemoveAll(tempdir)
+	require.NoError(t, err)
+	err = ioutil.WriteFile(filepath.Join(tempdir, "test1.txt"), []byte("foo"), 0644)
+	require.NoError(t, err)
+	err = ioutil.WriteFile(filepath.Join(tempdir, "test2.txt"), []byte("foo"), 0644)
+	require.NoError(t, err)
+
+	opid, err := sfs.SimpleFSMakeOpid(ctx)
+	require.NoError(t, err)
+
+	err = sfs.SimpleFSList(ctx, keybase1.SimpleFSListArg{
+		OpID: opid,
+		Path: keybase1.NewPathWithLocal(tempdir),
+	})
+	require.NoError(t, err)
+
+	listResult, err := sfs.SimpleFSReadList(ctx, opid)
+	require.NoError(t, err)
```

Comment:
This will fail if you run this test for real, right?  I notice this directory's tests are not yet included in `appveyor.yml` or `Jenkinsfile`.  I assume that's on purpose until this test will actually pass?